vela-templates/gen_definitions.sh

```diff
@@ -66,12 +71,12 @@ function renderMinimal {
 }
 
 echo -e "${HEAD_PROMPT}Start generating definitions at ${LIGHTGRAY}${SCRIPT_DIR}${NC} ..."
-echo -ne "${HEAD_PROMPT}${YELLOW}(0/2) Generating internal definitions from ${LIGHTGRAY}${INTERNAL_DEFINITION_DIR}${YELLOW} to ${LIGHTGRAY}${INTERNAL_TEMPLATE_DIR}${YELLOW} ... "
+echo -ne "${HEAD_PROMPT}${YELLOW}(0/2) Generating internal definitions from ${LIGHTGRAY}${INTERNAL_DEFINITION_DIR}${YELLOW} and ${LIGHTGRAY}${DEPRECATED_DEFINITION_DIR}${YELLOW} to ${LIGHTGRAY}${INTERNAL_TEMPLATE_DIR}${YELLOW} ... "
 export AS_HELM_CHART=true
-render $INTERNAL_DEFINITION_DIR $INTERNAL_TEMPLATE_DIR
+render $DEPRECATED_DEFINITION_DIR $INTERNAL_DEFINITION_DIR $INTERNAL_TEMPLATE_DIR
 renderMinimal $INTERNAL_TEMPLATE_DIR $MINIMAL_TEMPLATE_DIR
 echo -ne "${GREEN}Generated.\n${HEAD_PROMPT}${YELLOW}(1/2) Generating registry definitions from ${LIGHTGRAY}${REGISTRY_DEFINITION_DIR}${YELLOW} to ${LIGHTGRAY}${REGISTRY_TEMPLATE_DIR}${YELLOW} ... "
 export AS_HELM_CHART=system
-render $REGISTRY_DEFINITION_DIR $REGISTRY_TEMPLATE_DIR
+render "" $REGISTRY_DEFINITION_DIR $REGISTRY_TEMPLATE_DIR
```

Comment:
Why not directly call `render ${DEPRECATED_DEFINITION_DIR} ${INTERNAL_TEMPLATE_DIR}` instead of rewrite function `render`?

Reply:
Because `func render` will delete all the defs first.